mlserver/batching/adaptive.py

```diff
@@ -75,9 +83,8 @@ def _start_batcher_if_needed(self):
                 # If task hasn't finished yet, let it keep running
                 return
 
-        self._batching_task = schedule_with_callback(
-            self._batcher(), self._batching_task_callback
-        )
+        self._batching_task = asyncio.create_task(self._batcher())
```

Comment:
What was the reason to replace the `schedule_with_callback` call? Under the hood, `schedule_with_callback` will essentially call the same things (it's essentially a helper to avoid duplicating these two lines).

Reply:
I didn't touch that part. It must be a matter of updating issue to the master.

Reply:
Is there any chance this was a leftover from the previous PR? Either way, could you try to rebase from `master` to see if it goes away?